pkgs/tools/virtualization/multipass/default.nix

```diff
@@ -15,17 +15,15 @@
 , pkg-config
 , qemu
 , qemu-utils
-, qtbase
-, qtx11extras
+, qt6
```

Comment:
Please use qt6Packages.callPackage in all-packages.nix.

Reply:
Ah this is what I was missing, thanks!